examples/pxScene2d/src/shell.js

```diff
@@ -175,7 +170,6 @@ if (false)
     scene.root.on("onKeyDown", function(e)
     {
       var code = e.keyCode; var flags = e.flags;
-      console.log("onKeyDown shell:", code, ", ", flags);
```

Comment:
For the time being, let's leave the key prints in place as they are useful for debugging.